cpp/ql/src/semmle/code/cpp/exprs/Assignment.qll

```diff
@@ -7,9 +7,9 @@ import semmle.code.cpp.exprs.BitwiseOperation
  * etc. A C++ overloaded assignment operation looks syntactically identical but is instead
  * a `FunctionCall`.
  *
- * This is an abstract root QL class for all (non-overloaded) assignments.
+ * This is an root QL class for all (non-overloaded) assignments.
```

Comment:
```suggestion
 * This is a QL base class for all (non-overloaded) assignments.
```

(_root_ means something specific in QL)

Reply:
Thank you! I will push the updated upgrade script as well, and then generate the db stats